src/components/intercom/index.module.css

```diff
@@ -3,15 +3,15 @@
   height: 36px;
   bottom: 6px;
   right: 6px;
-  min-width: 123px;
-  padding-left: 13px;
+  padding-left: 18px;
   padding-right: 18px;
   z-index: 15;
 }
 
 @screen md {
   .launcher {
+    min-width: 123px;
```

Comment:
that's a pretty random number :D how does it come to be?

Reply:
That number already existed, I didn't change anything..